api/maven-api-model/src/main/mdo/maven.mdo

```diff
@@ -3007,7 +3007,7 @@
         is the location of a file that needs to exist, and if it doesn't, the profile will be
         activated. On the other hand, {@code exists} will test for the existence of the file and if it is
         there, the profile will be activated.<br>
-        Variable interpolation for these file specifications is limited to {@code ${basedir}},
+        Variable interpolation for these file specifications is limited to {@code ${project.basedir}},
```

Comment:
Would it make sense to also support `session.rootDirectory` ?

Reply:
We can add - currently we have implementation for `rootDirectory` - but I would like change to `project.rootDirectory`
https://issues.apache.org/jira/browse/MNG-7897

I would like to cherry-pick this for 3.9.5 - so only one change in one PR